packages/bpk-svgs/readme.md

```diff
@@ -7,3 +7,17 @@
 ```sh
 npm install bpk-svgs --save-dev
 ```
+
+## Font usage
+We ship our icons as font as well, the mapping is available in `iconMapping.json`, to use it you can simply import the font and use it
+
+```
+import BpkIcon from 'bpk-svgs/dist/font/BpkIcon.ttf';
```

Comment:
So i know I used this as an example, but i didn't mean it literally :P

I mean you can configure `file-loader` in webpack or something which would allow users to import the font in Javascript, but it's not very standard, is it? 

I thought we would just list the paths to the font file and mapping json.

Open to other thoughts here.